ml-agents-envs/mlagents_envs/registry/binary_utils.py

```diff
@@ -38,10 +38,13 @@ def get_local_binary_path(name: str, url: str) -> str:
             break
         try:
             download_and_extract_zip(url, name)
-        except IOError:
-            logger.debug(
-                f"Attempt {attempt + 1} / {NUMBER_ATTEMPTS} : Failed to download"
-            )
+        except Exception:  # pylint: disable=W0702
```

Comment:
Is there a particular exception that is given here when it fails to download?

Reply:
A lot of things could go wrong : Could have internet problems, zip extraction problems, permission issues, IOErrors...

Reply:
I suppose ideally we only retry certain things, specifically internet/download problems. But that seems more difficult than it's worth

Reply:
Maybe we can just change the message to "Failed to download and extract"